<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>mountDialog</class>
 <widget class="QDialog" name="mountDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>582</width>
    <height>304</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>250</y>
     <width>341</width>
     <height>32</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>40</y>
     <width>121</width>
     <height>51</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QCheckBox" name="renewBox">
      <property name="text">
       <string>Enable renew</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QCheckBox" name="vstatBox">
      <property name="text">
       <string>Enable vstat</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>20</y>
     <width>111</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Mount Options</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="srcText">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>40</y>
     <width>361</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="dstText">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>110</y>
     <width>361</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>20</y>
     <width>51</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Source</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>90</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Destination</string>
   </property>
  </widget>
  <widget class="QPushButton" name="srcBrowseButton">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>70</y>
     <width>91</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>Browse</string>
   </property>
  </widget>
  <widget class="QPushButton" name="dstBrowseButton">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>140</y>
     <width>91</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>Browse</string>
   </property>
  </widget>
  <widget class="QComboBox" name="typeBox">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>240</y>
     <width>231</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>220</y>
     <width>111</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Mount Type</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="excpText">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>180</y>
     <width>361</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="excpBrowseButton">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>210</y>
     <width>91</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>Browse</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>160</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Exceptions</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>mountDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>mountDialog</receiver>
   <slot>Accepted()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>451</x>
     <y>271</y>
    </hint>
    <hint type="destinationlabel">
     <x>127</x>
     <y>278</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>srcBrowseButton</sender>
   <signal>clicked()</signal>
   <receiver>mountDialog</receiver>
   <slot>SrcBrowseClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>315</x>
     <y>82</y>
    </hint>
    <hint type="destinationlabel">
     <x>443</x>
     <y>105</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>dstBrowseButton</sender>
   <signal>clicked()</signal>
   <receiver>mountDialog</receiver>
   <slot>DstBrowseClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>341</x>
     <y>165</y>
    </hint>
    <hint type="destinationlabel">
     <x>428</x>
     <y>148</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>excpBrowseButton</sender>
   <signal>clicked()</signal>
   <receiver>mountDialog</receiver>
   <slot>ExcpBrowseClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>336</x>
     <y>218</y>
    </hint>
    <hint type="destinationlabel">
     <x>423</x>
     <y>205</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>Accepted()</slot>
  <slot>SrcBrowseClicked()</slot>
  <slot>DstBrowseClicked()</slot>
  <slot>ExcpBrowseClicked()</slot>
 </slots>
</ui>
